src/core/FieldUtils.H

```diff
@@ -26,10 +26,8 @@ inline amrex::IndexType index_type(const FieldLoc floc)
         return amrex::IndexType(amrex::IntVect::TheDimensionVector(2));
     }
 
-#ifdef AMREX_USE_GPU
```

Comment:
Just curious, why did you have to remove this? I had put this in because the explicit return value seemed only needed for GPU builds.

Reply:
The CI system wouldn't compile because it would complain that the function was not void and not returning a value and treated it as an error.

Reply:
Are you fine with this change? I can take it out and see if CI still passes. That's the only reason I changed it.

Reply:
It was really surprising it was an error and not a warning.

Reply:
Sorry, I am totally fine with this. I was just curious for the reason behind and now I understand.